converter/config/yaml/filtering.go

```diff
@@ -0,0 +1,59 @@
+package yaml
+
+import "net"
+import "github.com/pkg/errors"
+
+//filtering implements config.Filtering
+type filtering struct {
+	AlwaysInclude   []string `yaml:"AlwaysInclude"`
+	NeverInclude    []string `yaml:"NeverInclude"`
+	InternalSubnets []string `yaml:"InternalSubnets"`
+}
+
+func (f *filtering) GetAlwaysIncludeSubnets() ([]net.IPNet, []error) {
+	return f.parseSubnetList(f.AlwaysInclude)
+}
+
+func (f *filtering) GetNeverIncludeSubnets() ([]net.IPNet, []error) {
+	return f.parseSubnetList(f.NeverInclude)
+}
+
+func (f *filtering) GetInternalSubnets() ([]net.IPNet, []error) {
+	return f.parseSubnetList(f.InternalSubnets)
+}
+
+func (f *filtering) parseSubnetList(netList []string) ([]net.IPNet, []error) {
```

Comment:
Parses each string in netList as a full CIDR range. If the "/XX" part is missing, we try to parse the string as a IP address. If we had to parse the string as an IP address, we convert it into a CIDR range with a fully marked netmask.